rosdep/python.yaml

```diff
@@ -6273,6 +6280,9 @@ python3-mock:
   openembedded: [python3-mock@meta-ros-common]
   rhel: ['python%{python3_pkgversion}-mock']
   ubuntu: [python3-mock]
+python3-more-itertools:
+  debian: [python3-more-itertools]
+  ubuntu: [python3-more-itertools]
```

Comment:
could you add the following platforms as well?

- fedora `python3-more-itertools` https://fedora.pkgs.org/33/fedora-aarch64/python3-more-itertools-8.4.0-1.fc33.noarch.rpm.html
- arch `python-more-itertools` https://archlinux.org/packages/community/any/python-more-itertools/
- rhel `'python%{python3_pkgversion}-more-itertools'` https://centos.pkgs.org/8/epel-x86_64/python3-more-itertools-7.2.0-3.el8.noarch.rpm.html
- gentoo `dev-python/more-itertools` https://packages.gentoo.org/packages/dev-python/more-itertools